source/JointMilitarySymbologyLibraryCS/jmsml.config

```diff
@@ -161,5 +161,409 @@
       <AmplifierValue ID="REDUCED" Label="-">1</AmplifierValue>
       <AmplifierValue ID="REINFORCED_REDUCED" Label="+/-">2</AmplifierValue>
     </AmplifierValues>
+
+    <SchemaTypes>
```

Comment:
Good points all, @csmoore .

I actually thought that you DID deserve a reply to each of your comments, because you reviewed so well and so thoroughly, even if the comment is, "Noted, thanks for the input."  I am probably guilty of not replying to enough comments in the past.

But this all goes to show that what may be insufficient communication for one person, is more than enough to another person.  What may seem like a tedious task to perform for one, another thinks is vital to the way we all have to operate.  I think we all just need to accept that people have different approaches/opinions, be patient with each other, and accept that not everyone will be happy with everything you or I do or you or I don't do.  We all just do what we can to move the "whole" forward.